core/src/main/scala/com/salesforce/op/stages/impl/feature/Transmogrifier.scala

```diff
@@ -685,3 +690,43 @@ trait MapStringPivotHelper extends SaveOthersParams {
     OpVectorMetadata(outputName, cols, Transmogrifier.inputFeaturesToHistory(inputFeatures, stageName))
   }
 }
+
+
+/**
+ * Summary statistics of a text feature
+ *
+ * @param valueCounts counts of feature values
+ */
+private[op] case class TextStats(valueCounts: Map[String, Int]) extends JsonLike
```

Comment:
lets put `CategoricalDetection` into a separate file (together with `TextStats`

Reply:
Where should I put the file? In `com/salesforce/op/stages/impl/feature` ?

Reply:
sounds good.